packages/react-native-renderer/src/ReactNativeBridgeEventPlugin.js

```diff
@@ -66,7 +67,12 @@ function getParent(inst) {
 /**
  * Simulates the traversal of a two-phase, capture/bubble event dispatch.
  */
-export function traverseTwoPhase(inst: Object, fn: Function, arg: Function) {
+export function traverseTwoPhase(
```

Comment:
I'm not clear why this was exported, I couldn't find any usage beyond to update; but I might be missing something!

Reply:
I think my comment didn't save, so I'm sorry if I'm saying this twice - instead of passing a flag than makes this `traverseTwoPhase` function only traverse one phase, what do you think about creating a new function named `traverseSinglePhase` or `traverseCapturePhase` and calling that instead? It would be a bit of duplication, but in general a little duplication is fine because it will get inlined and optimized anyway.